kafka-connect/scripts/kafka_connect_run.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+set -e
+
+# if command starts with an option, prepend qdrouterd
+if [ "$1" = 'bash' ]; then
+    exec "$@"
+else
```

Comment:
what are this three lines about qdoruterd ?

Reply:
@ppatierno Ehm, that looks like copy paste error in the comment :-| (fixed now) ... but the code it self is OK. It allows you to request bash when starting the Docker container with `docker run -ti enmassepriject/kafka-connect:latest bash`. That is useful for debugging etc.